### DISCUSSION TITLE
怎么让配置左侧菜单收缩起来，我发现只要我一点其他菜单页面加载后左侧菜单就全部展开了，有没有什么配置项目可以做到。

### DISCUSSION THREAD

**Discussion by mengzhi139** [vbenjs/vue-vben-admin#2871] [Q&A]
// 在路由切换的时候关闭左侧混合菜单展开菜单
    closeMixSidebarOnChange: false,

这个我已经测试过了，好像解决不了问题

**Comment by mengzhi139** [comment]
![image](https://github.com/vbenjs/vue-vben-admin/assets/57259962/bc9fc0a3-ead2-4712-b642-2b6c6549e9ab)
我看DEMDEMO上是都收起来的，但是我的是展开的，这样页面会拉很长，但是没有数据填充

**Comment by Novices666** [comment]
我现在用的V5也是这个情况，解决了码？
